### PR TITLE
Make strobe for alert work

### LINKED ISSUE #64: /Pub alert strobe doesn't trigger
/Pub alert strobe doesn't trigger because `.get()` is never called. Furthermore we might not want to call it from both the IRC and the Slack bot.

### PR DESCRIPTION
**Pull request by jabbink**
fixes #64

### CODE REVIEW

**Comment by RickWieman** [comment]
This still won't work

File: scripts/pubtimealert.coffee [hunk 1]
```diff
@@ -25,8 +25,8 @@ module.exports = (robot) ->
   pubtime = new cronJob PUB_TIME,
     ->
       robot.messageRoom ROOM, "The /Pub is open! Have a beer! :beers:"
-      #implement the CHue strobe here....
       robot.http("#{chueURL}strobe/#{lamp}" + duration)
```
Comment (RickWieman) [inline scripts/pubtimealert.coffee]: There's a slash missing here

**Comment by jabbink** [comment]
Now it will.

**Comment by RickWieman** [comment]
Cool. The only thing from #64 it won't fix is the fact that this will fire once for every instance. IRC and Slack are two different instances, so it will fire the strobe twice.

**Pull request merged**